proto/brpb.proto

```diff
@@ -347,8 +348,21 @@ message BackupResponse {
     kvrpcpb.APIVersion api_version = 5;
 }
 
+message CheckAdminRequest {
+}
+
+message CheckAdminResponse {
+    Error error = 1;
+    metapb.Region region = 2;
+    bool has_pending_admin = 3;
+}
+
 service Backup {
     rpc backup(BackupRequest) returns (stream BackupResponse) {}
+    // CheckAdmin used for snapshot backup. before we start snapshot for a TiKV. 
+    // we need stop all schedule first and make sure all in-flight schedule has finished.  
+    // this rpc check all pending conf change for leader.
+    rpc CheckAdmin(CheckAdminRequest) returns (stream CheckAdminResponse) {}
```

Comment:
name it to HasRegionSchedule?

Reply:
I think schedule is also not clear, we need check split/merge rather than schedule 🤔
CheckPendingAdmin maybe better?

Reply:
or `CheckPendingAdminOp`, don't know what it mean by `pending admin`

Reply:
CheckPendingAdminOp is ok for me, WDYT? cc @hicqu

Reply:
change it to CheckPendingAdminOp for now, if it's not ok, we can change it in future.